src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,198 @@
+    <br /><img src="" alt="Extension showing the cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
+2. Select the *Budget* tab.
+    <br /><img src="" alt="Budget tab with remaining balance and option to change budget amount" style="max-width:300px" class="img-outline"/>
+3. Enter the new amount in the **Budget amount** field.
+4. Select the **Monthly** toggle to enable/disable monthly renewals of your budget.
+5. Click **Submit changes**. A new tab opens to your wallet provider. 
+6. Sign in to your account if you haven't already, then confirm the new budget amount.
+</Steps>
+
+## Change your continuous payment settings
+
+A continuous payment is a payment that recurs at your set rate for as long as you are active on a web monetized page. When you leave the page, the payments stop.
+
+Continuous payments are enabled by default and sent at a rate of $0.60 USD per hour. Use the instructions below to change the rate or disable continuous payments. 
+
+<Steps>
+1. Click the cog in the upper-right of the extension.
+    <br /><img src="" alt="Extension showing the cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
+2. Select the *Rate* tab.
+    <br /><img
+    src=""
+    alt="Rate tab with rate of pay field and continuous payment toggle"
+    style="max-width:300px"
+    class="img-outline"
+    />
+3. Enter the amount the extension will send, per hour, to web monetized sites.
+4. Select the **Continuous payment** toggle to enable/disable continuous payments. This is a global setting that applies to all web monetized pages you visit.
+</Steps>
+
+## Make a one-time payment 
+
+You can make a one-time payment (OTP) to a page instead of, or in addition to, continuous payments. The amount of the OTP is deducted from your budget. 
+
+<Steps>
+1. Ensure the URL shown above the amount field is the page you intend to pay. 
+    <br /><img
+    src=""
+    alt="Extension showing an available balance of five dollars, with one dollar in the amount field and send now button available"
+    style="max-width:300px"
+    class="img-outline"
+    />
+2. Enter the amount and click **Send now**. The button's text will briefly change to *Payment successful*, then back to *Send now*.
+    <br /><img
+    src=""
+    alt="Extension showing an available balance of four dollars, with the amount field set to zero and payment successful showing in button"
+    style="max-width:300px"
+    class="img-outline"
+    />
+</Steps>
+
+## Resolve a wallet connection error
+
+When you try to connect the extension to your digital wallet, the extension attempts to automatically connect with your wallet provider to add a key to your wallet account. 
+
+You'll receive an error if the connection fails. When this happens, you must copy the key that appears in the extension and manually add it to your wallet account. The steps for adding the key to your account depend on who your wallet provider is. 
```

Comment:
We're only addressing key addition failure as error here. There could be other errors as well? Maybe should update text to show chances that it might be due to key addition failure or something else and then explain the manual key addition part?

FYI, with key add failure, user gets redirected to [this page](https://webmonetization.org/welcome/?result=key_add_error&intent=connect), although the text for error is same as any other failure (can change if needed, the URL has "key_add_error").

Reply:
We need more descriptive error messages. Telling the user something went wrong, but not telling them what to try to fix the issue is no bueno. Also, if the error messages are descriptive enough, there's no need to document them in the user docs. 

For now, I'm going to change the section title from "Resolve a wallet connection error" to something more specific to "key addition failure".

Reply:
> Also, if the error messages are descriptive enough, there's no need to document them in the user docs.

I agree. (Similarly with those 6 extension icons)